src/api/controller/front.js

```diff
@@ -68,8 +68,8 @@ const getDefaultInitialState = (token, cookie, locale) => ({
         locale: locale,
         phrases:
             locale === 'fr' || locale === 'fr-FR'
```

Comment:
Why not take the two first letters of the `locale`?
Here is the list of possibilities:

- fr-BE
- fr-CA
- fr-CH
- fr-FR
- fr-LU
- fr-MC

Reply:
You are right, I use one solution found in another file but better is to check only the two first letters.